index.html

```diff
@@ -12,14 +12,73 @@
 
 <body>
   <div class="main">
-    <h2 class="title">Front-End Tools</h2>
-    <div class="card">
-      <img class="card-image" src="assets/images/react_icon.svg" alt="React icon">
-      <h3 class="card-title">ReactJS</h3>
-      <p class="card-release">Release: 2013</p>
-      <p class="card-info">
-        React is a declarative, component-based JavaScript library for building user interfaces.
-      </p>
+    <h2 class="title">Front-end Tools</h2>
+    <div class="all-cards">
+      <div class="card">
+        <img class="card-image" src="assets/images/react_icon.svg" alt="React icon">
+        <h3 class="card-title">ReactJS</h3>
+        <p class="card-release">Release: 2013</p>
+        <p class="card-info">React is a declarative, component-based Javascript library for building user interfaces.</p>
+      </div>
+      <div class="card">
+        <img class="card-image" src="assets/images/angular_icon.svg" alt="React icon">
+        <h3 class="card-title">AngularJS</h3>
+        <p class="card-release">Release: 2010</p>
+        <p class="card-info">AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building
```

Comment:
Change this for something as this:
```suggestion
        <p class="card-info">
          AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.
        </p>
```